# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/nsIEditor.idl
# Commit: d3816f82c0b9
# Full Hash: d3816f82c0b9d4ceca0bb2ad177694164ed0921f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-03 21:40:10
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r 876b1031a6ae -r d3816f82c0b9 editor/nsIEditor.idl
--- a/editor/nsIEditor.idl	Fri Apr 03 01:38:22 2020 +0000
+++ b/editor/nsIEditor.idl	Fri Apr 03 01:39:36 2020 +0000
@@ -261,6 +261,7 @@
     * Calls to beginTransaction can be nested, as long as endTransaction
     * is called once per beginUpdate.
     */
+  [can_run_script]
   void beginTransaction();
 
   /** endTransaction is a signal to the editor that the caller is